lib/tasks/sync_canonical_models.rake

```diff
@@ -123,14 +123,17 @@ namespace :canonical_models do
       Rails.logger.info 'Syncing canonical materials...'
 
       args.with_defaults(preserve_existing_records: false)
+      preserve_existing_records = FALSEY_VALUES.exclude?(args[:preserve_existing_records])
```

Comment:
There is a [card](https://trello.com/c/K8o44aNI/170-refactor-and-add-tests-for-rake-tasks-syncing-canonical-models) to refactor and add tests for these Rake tasks, so for now I'm letting them be ugly.